aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -96,6 +96,8 @@ module aptos_token::token {
         default_properties: PropertyMap,
         //control the TokenData field mutability
         mutability_config: TokenMutabilityConfig,
+        // content hash of the media
+        content_hash: String,
```

Comment:
Should we introduce a `Hash` type in the framework which has a 256 bit size? We discussed in some recent Move community meetings that independent of a native u256 type, such a hash type seems useful. You don't provide numeric algebra on hashes, so the set of operations is much smaller. We can use a relative bare bones Move `Hash` type for now, and then extend it over time. Basic point is whether we can agree on 256 bit precision.

It should probably be added to the `aptos_hash` module. The representation can be two `u128`, or whatever else division makes sense. This type would have just

Reply:
sounds great. will try to modify the aptos_hash to introduce this `hash` type.